### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by brownleeb** [VirusPilot/stratux-pi4#12] [Q&A]
How do you recommend updating?  Just do a full pull and re-make?  I was updating individual files as I saw they changed, but the updates last month were just too many so i did a rebuild.
I have been using the pi4-viruspilot version for a few months and its great.  I actually had it built and running on Arch linux (pi4) but had some minor issues with wifi so loaded up raspbian.

**Comment by VirusPilot** [comment]
I would indeed do a pull and make && make install.
